### PR TITLE
Client is rendering & fetching data once URI is changed

### PR DESCRIPTION
**Pull request by jjdev86**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
 ## API
-  ## GET all questions/answers per productId
-/questions/:productId
+  ## GET all questions per product
+/products/:productId
 
-  ## POST question
-/questions/add
+succesful data:
+```sh
```
Comment (Bruce773) [inline README.md:27]: I love all the detail you put into your README!!

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,20 +1,180 @@
...
 
   render() {
+    const { product } = this.state;
+    // questios array
+    const data = this.state.product.questions;
     return (
       <div>
-        <h1>Questions will go here</h1>
-        <div>{this.state.test}</div>
+        <div id="ask-btf_feature_div">
```
Comment (Bruce773) [inline client/src/App.jsx:72]: Have you thought about making wrappers that contain these divs and return their child components? Seems like there are a lot of divs in the App component. Usually, you want App to just contain React components and you let those components handle what divs they need.

File: client/src/Foo.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
+
```
Comment (Bruce773) [inline client/src/Foo.js]: Is this an example component?

File: client/src/components/Search.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+		super(props);
+		this.state = {
+			search: '',
+			searchResult: []
+		}
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	// updates the search value
+	handleChange(e) {
```
Comment (Bruce773) [inline client/src/components/Search.jsx]: Dude! Nice work on this component!! 
My only suggestion would be... could you run Prettier over this file?

**Review by Bruce773 — COMMENTED** [review]
Dude!! Awesome work!! 💯 😄

File: client/src/components/Search.jsx (hunk 4 above)
Comment (jjdev86) [reply client/src/components/Search.jsx]: ran prettier. Thanks!

File: client/src/Foo.js (hunk 3 above)
Comment (jjdev86) [reply client/src/Foo.js]: An example that has been removed.

File: client/src/App.jsx (hunk 2 above)
Comment (jjdev86) [reply client/src/App.jsx:72]: I'm not too sure on how to use wrappers :( but thanks tho.

**Pull request merged**